dev-requirements.txt

```diff
@@ -11,6 +11,4 @@ nbconvert==6.0.6
 nbsphinx==0.8.5
 Sphinx==3.2.1
 pydata_sphinx_theme==0.4.0
-nlp_primitives>=1.0.0
```

Comment:
I think we can remove the `featuretools-sklearn-transformer` requirement at the top of this file as well.

Reply:
Fixed